addons/purchase_stock/models/purchase.py

```diff
@@ -227,6 +227,10 @@ class PurchaseOrderLine(models.Model):
     move_ids = fields.One2many('stock.move', 'purchase_line_id', string='Reservation', readonly=True, ondelete='set null', copy=False)
     orderpoint_id = fields.Many2one('stock.warehouse.orderpoint', 'Orderpoint')
     move_dest_ids = fields.One2many('stock.move', 'created_purchase_line_id', 'Downstream Moves')
+    propagate_date = fields.Boolean(string="Propagate Rescheduling", default=True,
```

Comment:
why default true here and not in mrp?

Reply:
so as for the stock move, do not set defaults to these fields, their values come from the stock rule else they are empty